packages/webdriverio/src/utils/interception/devtools.ts

```diff
@@ -100,7 +107,9 @@ export default class DevtoolsInterception extends Interception {
                 /**
                  * no stubbing if no overwrites were defined
                  */
-                if (mock.respondOverwrites.length === 0) {
+                if (mockResponded || mock.respondOverwrites.length === 0) {
+                    mock.emit('continue', requestId)
+
                     continue
```

Comment:
emitting `continue` if network response already was sent (so we could be sure at least some of `fail` / `overwrite` / `continue` will be emitted per `request`. With this implementation we could also know if `respond-mock` was ignored because another `respond-mock` already overwrited the response. The example is [here](https://github.com/webdriverio/webdriverio/pull/10137/files#diff-969d2ca1082ce6766dd3fd8e3cf4f967cf6aeaef0640b11fbadb975381f85597R581-R590)).